From 86be611fe24966eec9f364b419b3ca44697504c8 Mon Sep 17 00:00:00 2001
From: Sebastian Reichel <sre@kernel.org>
Date: Sat, 6 Jun 2026 01:04:49 +0200
Subject: [PATCH] arm64: dts: qcom: x1-el2: Drop Add remoteproc IOMMUS

Having the remoteproc iommus described results in massive IOMMU errors
and boot failure for me.

Signed-off-by: Sebastian Reichel <sre@kernel.org>
---
 arch/arm64/boot/dts/qcom/x1-el2.dtso | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/qcom/x1-el2.dtso b/arch/arm64/boot/dts/qcom/x1-el2.dtso
index 1677d902d24d3a..e04ef465199e20 100644
--- a/arch/arm64/boot/dts/qcom/x1-el2.dtso
+++ b/arch/arm64/boot/dts/qcom/x1-el2.dtso
@@ -59,12 +59,12 @@
 };
 
 &remoteproc_adsp {
-	iommus = <&apps_smmu 0x1000 0x80>;
+	//iommus = <&apps_smmu 0x1000 0x80>;
 	qcom,broken-reset;
 };
 
 &remoteproc_cdsp {
-	iommus = <&apps_smmu 0x0c00 0x0>;
+	//iommus = <&apps_smmu 0x0c00 0x0>;
 	qcom,broken-reset;
 };
 
